CONTRIBUTING.md

```diff
@@ -96,4 +96,3 @@ guide.
 
 Please also enable **“Allow edits by maintainers”**, this will help to speed-up the review
 process as well.
-
```

Comment:
The last line should be empty, like it was

Reply:
This is still wrong, the last part of the file should just be reverted to how it was

Reply:
@spydon do we want one empty line in the end or two?

Cuz generally the files ends with one empty line (i.e. new line) and Github diff preview does not show the last new line. But the file originally had two empty new lines at end and I removed the extra empty/new line.

Reply:
<img width="1062" alt="Screenshot 2023-12-22 at 8 12 30 PM" src="https://github.com/invertase/melos/assets/74326345/3da0761e-ca2f-4d23-8be7-bfa7adfc8ae9">

Here the left file has no empty/new line at the end and the right one have 1 empty/new line at the end. But as you can see in the image above, Github does not preview the last empty line, just the `-` in red circle if new line does not exist at the end of file.

Reply:
Aah right, it should just be one.